compose/install.md

```diff
@@ -59,6 +60,14 @@ To install Compose, do the following:
 
 ## Alternative install options
 
+### Install using apt-get on Ubuntu
+
+Compose can be installed on Ubuntu using the following command:
+
+```bash
+$ sudo apt-get install docker-compose
+```
```

Comment:
I don't think we should suggest this method of install. The package is not provided by Docker, and it is not well maintained (version is version old).